static/app/views/performance/newTraceDetails/traceDrawer/details/errorNodeDetails.tsx

```diff
@@ -0,0 +1,105 @@
+import styled from '@emotion/styled';
+
+import {Button} from 'sentry/components/button';
+import {
+  SpanDetailContainer,
+  SpanDetails,
+} from 'sentry/components/events/interfaces/spans/newTraceDetailsSpanDetails';
+import {DataSection} from 'sentry/components/events/styles';
+import {generateIssueEventTarget} from 'sentry/components/quickTrace/utils';
+import {IconFire} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import type {Organization} from 'sentry/types';
+import {Row} from 'sentry/views/performance/traceDetails/styles';
+
+import type {TraceTree, TraceTreeNode} from '../../traceTree';
+
+export default function ErrorNodeDetails({
```

Comment:
lets stick to named exports
```suggestion
export function ErrorNodeDetails({
```